_conferences/droidcon-uk-2017.md

```diff
@@ -5,4 +5,7 @@ location: London, UK
 
 date_start: 2017-10-26
 date_end:   2017-10-27
+
+ cfp_end:   2017-05-14 
```

Comment:
@JakeWharton Perhaps we should change the code so that it works with only `cfp_end` specified? i.e. if ((cfp_start is missing or before today) and (cfp_end is after today))